src/lando/settings.py

```diff
@@ -24,6 +24,10 @@
     "django-insecure-26k#ouat@%d6w5gmuhvo_vc=_@on^6=eh9*g!p-k9ynjvyc#(_",
 )
 
+ENCRYPTION_KEY = os.getenv(
```

Comment:
Nit: you can just use `os.getenvb` here instead of manually encoding.

Reply:
I think it's simpler to keep this as it is, since using `getenvb` would result in having to change all the strings to bytes fields, and would also be the only case where this is done in the settings file. I'd rather keep all the "keys" as strings, and the values as whatever type they need to be, to keep things simple. We already convert types elsewhere, and this is basically just doing that.